code/src/HappyForm.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react'
+import './happyForm.css'
+
+export const HappyForm = ({ onFormSubmit }) => {
+    const [message, setMessage] = useState('')
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        onFormSubmit(message)
+
+    }
+
+    return (
+        <form>
+            <h5>Post a happy thought!</h5>
+            <textarea
+                rows='3'
```

Comment:
When working on a project try to use `'` or `"` - do not combine them if you don't need to. In this case - you can use `"` as you are using all over the project
```suggestion
                rows="3"
```